homeassistant/components/camera/rpi_camera.py

```diff
@@ -26,6 +26,7 @@
 CONF_TIMELAPSE = 'timelapse'
 CONF_VERTICAL_FLIP = 'vertical_flip'
 
+DEFAULT_FILE_PATH = ''
```

Comment:
Why do we need a default?

Reply:
I added it prematurely. It was supposed to point somewhere else to prevent the files from being saved in the camera components folder.